client/src/api/user-api.js

```diff
@@ -0,0 +1,20 @@
+import api from "/api";
+
+async function createUser(auth0_id, controller) {
+  return api(
+    {
+      method: "post",
+      endpoint: "/create",
```

Comment:
should be `/user/create`

Reply:
why is it /user/create? in the routes we have:
```router.post("/create", UserController.createUser);```

Reply:
The server directs every /user request to the UserController, then the controller determines which requests to run based on the following subpath
See server routes/index.TA